openbas-api/src/main/java/io/openbas/rest/scenario/ScenarioImportApi.java

```diff
@@ -34,14 +43,30 @@ public ImportPostSummary importXLSFile(@PathVariable @NotBlank final String scen
         return injectService.storeXlsFileForImport(file);
     }
 
+    @PostMapping(SCENARIO_URI + "/{scenarioId}/xls/{importId}/dry")
+    @Transactional(rollbackOn = Exception.class)
+    @Operation(summary = "Test the import of injects from an xls file")
+    @Secured(ROLE_USER)
+    public ImportTestSummary dryRunImportXLSFile(@PathVariable @NotBlank final String scenarioId,
+                                               @PathVariable @NotBlank final String importId,
+                                               @Valid @RequestBody final InjectsImportInput input) {
+        // Getting the mapper to use
+        ImportMapper importMapper = importMapperRepository
+                .findById(UUID.fromString(input.getImportMapperId()))
+                .orElseThrow(() -> new ElementNotFoundException(String.format("The import mapper %s was not found", input.getImportMapperId())));
+
+        return injectService.importInjectIntoScenarioFromXLS(scenarioId, importMapper, importId, input.getName(), input.getTimezoneOffset(), false);
+    }
+
     @PostMapping(SCENARIO_URI + "/{scenarioId}/xls/{importId}/test")
```

Comment:
And this one the dry run no ?

Reply:
The way I see it is : 
/dry when you want to do a dry run right before importing for good
/test when you want to test the mapper itself
I'm fine with changing the names though if needed

Reply:
It's ok for me